backend/local/fadvise_unix.go

```diff
@@ -77,12 +100,36 @@ func (f *fadvise) freePagesIfNeeded() {
 }
 
 func (f *fadvise) freePages() {
-	if err := unix.Fadvise(f.fd, f.lastPos, f.curPos-f.lastPos, unix.FADV_DONTNEED); err != nil {
-		fs.Debugf(f.o, "fadvise dontneed failed on file descriptor %d: %s", f.fd, err)
+	p := offsetLength{f.lastPos, f.curPos - f.lastPos}
+	select {
+	case f.freePagesCh <- p:
+	default:
+		fs.Debugf(f.o, "Blocked on queueing pages to clean for file descriptor %d", f.fd)
```

Comment:
I discovered that for certain type of copy this debug was quite noisy!  Do we need it?  Can we just write to the channel not in a select?

Maybe we should make the channel a little bigger - maybe 16 entries?

Reply:
After experimentation I'd suggest sth like 64. I can sure skip the log.

Reply:
SGTM